src/components/container/container/container.ts

```diff
@@ -0,0 +1,110 @@
+import { CSSResultGroup, html, LitElement, TemplateResult } from 'lit';
+import { customElement, property } from 'lit/decorators.js';
+
+import { ConnectedAbortController } from '../../core/eventing';
+
+import style from './container.scss?lit&inline';
+
+/**
+ * It displays its content with the default page spacing.
+ *
+ * @slot - Use the unnamed slot to add anything to the container.
+ */
+@customElement('sbb-container')
+export class SbbContainerElement extends LitElement {
+  public static override styles: CSSResultGroup = style;
+
+  /** Whether the container is expanded. */
+  @property({ type: Boolean, reflect: true }) public expanded = false;
+
+  /** Variant of the container, like transparent, white etc. */
+  @property({ reflect: true }) public variant: 'transparent' | 'white' | 'milk' | 'midnight' =
+    'transparent';
+
+  private _abort = new ConnectedAbortController(this);
+
+  public override connectedCallback(): void {
+    super.connectedCallback();
+    const signal = this._abort.signal;
+
+    // Once the component is done rendering the container should find the first scrollable element
+    // This is necessary in case the container itself, or any parent element has a fixed height
+    super.getUpdateComplete().then(() => {
+      this._handlePageScroll();
```

Comment:
Discussion point:
Shouldn't the sticky bar animation be the responsibility of the sticky bar component?